mindjunkies/forums/models.py

```
@@ -0,0 +1,85 @@
+    content = models.TextField()
+    author = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, related_name='forum_replies')
+    parent_reply = models.ForeignKey('self', on_delete=models.CASCADE, null=True, blank=True,
+                                     related_name='child_replies')
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    # Simple reaction counter
+    like_count = models.PositiveIntegerField(default=0)
+
+    class Meta:
+        ordering = ['created_at']
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)